pages/dao/[symbol]/proposal/[pk].tsx

```diff
@@ -58,17 +111,31 @@ const Proposal = () => {
                 <ProposalStateBadge
                   proposalPk={proposal.pubkey}
                   proposal={proposal.info}
+                  open={true}
                 />
               </div>
               <ProposalTimeStatus proposal={proposal?.info} />
             </div>
+
             {description && (
               <div className="pb-2">
                 <ReactMarkdown className="markdown">
                   {description}
                 </ReactMarkdown>
               </div>
             )}
+
+            {signatoryRecord &&
+              showSignOffModal &&
+              (proposal.info.state === ProposalState.Draft ||
+                proposal.info.state === ProposalState.SigningOff) && (
+                <SignOffProposal
```

Comment:
I was not able to signOff any proposals. Always get an error 
<img width="911" alt="image" src="https://user-images.githubusercontent.com/5817016/146167333-fe5fffa6-6be1-4b2b-b717-f7d890b727b1.png">